strongbox-web-core/src/test/resources/etc/conf/strongbox-authentication-providers.yaml

```diff
@@ -0,0 +1,46 @@
+strongbox:
```

Comment:
why this class was moved?

Reply:
No idea. Also, this is a test resource.

Reply:
to use single test context in s-w-c. It wasn't moved, but used for tests in s-w-c. It still needs some work.